ste/uploader-blobFS.go

```diff
@@ -25,13 +25,56 @@ import (
 	"fmt"
 	"net/url"
 	"os"
+	"sync"
 	"time"
 
 	"github.com/Azure/azure-pipeline-go/pipeline"
 	"github.com/Azure/azure-storage-azcopy/azbfs"
 	"github.com/Azure/azure-storage-azcopy/common"
 )
 
+const bfsTrackerThreshold = (1024 ^ 3) * 10 // 10 GB
```

Comment:
Let's go for 20 GB. (10 GB is only 10 seconds worth or so, at our target speeds).